src-electron/zcl/zcl-loader-silabs.js

```diff
@@ -31,6 +31,10 @@ const dbEnum = require('../../src-shared/db-enum')
 const zclLoader = require('./zcl-loader')
 const _ = require('lodash')
 
+const fabricIndexFieldId = 0xfe
+const fabricIndexFieldName = 'FabricIndex'
+const fabricIndexType = 'fabric_idx'
+
```

Comment:
These need to be externalized somehow. We need to data-drive this via `zcl.json` somehow. There should be a field:
  "enableAutomaticFieldsLogic=true"
  which would turn this on, then these would be listed there as well, etc. You get the point.

Reply:
Yes, I agree.  Filed https://github.com/project-chip/zap/issues/629 for that.